src/ripple/protocol/SField.h

```diff
@@ -273,13 +273,7 @@ struct TypedField : SField
     using type = T;
 
     template <class... Args>
-    explicit TypedField(Args&&... args) : SField(std::forward<Args>(args)...)
-    {
-    }
-
-    TypedField(TypedField&& u) : SField(std::move(u))
-    {
-    }
+    explicit TypedField(private_access_tag_t pat, Args&&... args);
```

Comment:
Do we still need the `explicit` keyword? Won't the `private_access_tag_t` prevent the compiler from using implicit type conversion?

Reply:
Not if the parameter pack is empty, which it likely never will be in practice, but `explicit` doesn't hurt.

Reply:
hmm, that's crazy, i didn't know that! that pulls the rug under the purpose of having the private tag :)

Reply:
There's no real motivation to allow implicit construction as it is only constructed in one file, and even that is handled by a macro.